### PR TITLE
Move Analytics Builder doc into the Streaming Analytics guide

### PR DESCRIPTION
**Pull request by kailol-sag**
- The Analyltics Builder doc has been migrated from DITA to Markdown and included in the Streaming Analytics guide.
- The Streaming Analytics guide has been restructured so that there are separate bundles for Analytics Builder, EPL Apps, and the info that pertains to both.
- The sources for the Streaming Analytics guide are now in a new folder called "streaming-analytics".
- The existing "apama" folder has been removed. The links in the other guides still need to be adapted, but I plan to do this on a separate PR.
- Added redirects/aliases on bundle level as advised by Beate.
- Added many shortcodes to the Analytics Builder doc.
- Any fine-tuning to the contents can be done on a later PR.
- This PR also includes the block reference that is generated out of the developers' sources. Always to be replaced with the final version at the end of a development cycle.

This all needs to be backported into 10.18.

### CODE REVIEW

**Review by skom-sag — APPROVED** [review]

**Comment by MWindel** [comment]
@kailol-sag There are several instances, where there is a bulleted list with only 1 entry. As we do not use bulleted lists for that, please remove the list formatting. I have not commented at every occurrence.

File: content/streaming-analytics/epl-apps-bundle/advanced-features.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+```java
+send Event(..., new dictionary<string,any>) to Event.SEND_CHANNEL;
+send Event(..., {"fragment":<any>"value"}) to Event.SEND_CHANNEL;
+```
+
+The `MeasurementValue` type is provided for the measurements in the `Measurement` type. `MeasurementValue` has `value` and `unit` fields and `params` for other fragments.
+
+Example 1:
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/advanced-features.md:18]:
```suggestion
**Example 1**:
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+{{< /c8y-admon-caution >}}
+
+<a name="epl-apps"></a>
+#### Developing apps with the Streaming Analytics application
+
+The **EPL Apps** page of the Streaming Analytics application provides an interface for interactively editing new or existing EPL apps (\*.mon files) as well as importing and activating (deploying) EPL apps.
+
+Any user on the tenant wishing to use the **EPL Apps** page must be a **CEP Manager**. See [Administration > Managing permissions](/users-guide/administration/#managing-permissions) in the *User guide*.
+
+**Step 1 - Invoke the Streaming Analytics application**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:33]:
```suggestion
##### Step 1 - Invoke the Streaming Analytics application
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+
+* Export EPL apps. Use the **Export** command from the actions menu to download the app as a \*.mon file.
+
+* Deploy existing EPL apps. On the card that is shown for an app, change the mode from **Inactive** to **Active**. For more information, see [Deploying apps](/streaming-analytics/epl-apps/#deploying-apps).
+
+    When activating an app, any syntax errors are reported back immediately. The error state is shown on the card, helping you to ensure your app is in good shape. Click on the error to display information on what went wrong. It is not possible to activate an app if it has syntax errors. The errors are shown on the card until they have been fixed and the app has been activated again.
+
+* Reload all EPL apps. Click **Reload** in the top menu bar to refresh the display to show any changes other users have made since the page loaded, including any errors that have been introduced in the meantime.
+
+**Step 2 - Create an EPL app**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:59]:
```suggestion
##### Step 2 - Create an EPL app
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 4]
```diff
@@ -0,0 +1,785 @@
...
+Be aware that the EPL editor makes use of a standard web component. It provides many generic developer functions, some of which are not relevant to EPL, including but not limited to Quick Fix and Show Hover.
+{{< /c8y-admon-info >}}
+
+Click **X** in the top menu bar to leave the EPL editor and thus to return to the list of EPL apps.
+
+{{< c8y-admon-caution >}}
+All unsaved changes are lost when you navigate to a different URL or close the browser window.
+{{< /c8y-admon-caution >}}
+
+**Step 3 - Test the EPL app**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:83]:
```suggestion
##### Step 3 - Test the EPL app
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 5]
```diff
@@ -0,0 +1,785 @@
...
+Once your app is activated, you should be able to see the results of it running. This may include sending measurements, receiving data, creating alarms, and logging in the Apama-ctrl microservice. For information on how to check the log files of the Apama-ctrl microservice, see [Administration > Managing and monitoring microservices](/users-guide/administration/#managing-microservices) in the *User guide*.
+
+See also [Deploying apps](/streaming-analytics/epl-apps/#deploying-apps).
+
+<a name="sag-designer"></a>
+#### Developing apps with {{< sag-designer >}}
+
+{{< sag-designer >}} provides a full development environment and is the tool of choice when you have a complex EPL application. When your EPL app (that is, the monitor file) is ready, you must import it into {{< product-c8y-iot >}}.
+
+**Step 1 - Install Apama**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:94]:
```suggestion
##### Step 1 - Install Apama
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 6]
```diff
@@ -0,0 +1,785 @@
...
+
+If you have licensed Apama, install it using {{< sag-installer >}}.
+
+You can also use a freemium version of Apama which runs with reduced capabilities and several restrictions. If you want to use this, download the Apama Community Edition from [https://www.apamacommunity.com/downloads/](https://www.apamacommunity.com/downloads/) and install it.
+
+Both versions, the licensed version and the freemium version, include {{< sag-designer >}}.
+
+**Step 2 - Create a project**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:102]:
```suggestion
##### Step 2 - Create a project
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 7]
```diff
@@ -0,0 +1,785 @@
...
+
+Once installed, create an Apama project in {{< sag-designer >}} and enable it for {{< product-c8y-iot >}} connectivity. For instructions on how to create an Apama project, refer to [Creating Apama projects]({{< link-apama-webhelp >}}index.html#page/pam-webhelp%2FWIZARD_NEW_APAMA_PROJECT.html) in the Apama documentation.
+
+**Step 3 - Add Apama bundles to the project**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:106]:
```suggestion
##### Step 3 - Add Apama bundles to the project
```

File: content/streaming-analytics/epl-apps-bundle/basic-functionality.md [hunk 8]
```diff
@@ -0,0 +1,785 @@
...
+
+**Step 4 - Create a monitor file**
+
+To create a new Apama monitor file, refer to [Creating new monitor files for EPL applications]({{< link-apama-webhelp >}}index.html#page/pam-webhelp%2FWIZARD_NEW_MONITORSCRIPT.html) in the Apama documentation.
+
+Before you import the newly created monitor file as an EPL app into {{< product-c8y-iot >}} and activate it there, you might want to test if the monitor file works as expected from within {{< sag-designer >}}.
+
+For further information, see [The Cumulocity IoT Transport Connectivity Plug-in]({{< link-apama-webhelp >}}index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_the_cumulocity_connectivity_plug_in.html) in the Apama documentation.
+
+**Step 5 - Run and test the monitor file**
```
Comment (MWindel) [inline content/streaming-analytics/epl-apps-bundle/basic-functionality.md:138]:
```suggestion
##### Step 5 - Run and test the monitor file
```

File: content/streaming-analytics/block-reference-bundle/aggregate.md [hunk 9]
```diff
@@ -0,0 +1,985 @@
...
+<td><a href="#standard-deviation">Standard Deviation</a></td>
+<td><span>Calculates the standard deviation and variance of the values over time.</span>
+</td>
+</tr>
+</tbody>
+</table>
+
+### Average (Mean)
+
+**apama.analyticskit.blocks.core.Mean**
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/block-reference-bundle/aggregate.md:66]: @MWindel , this is part of the block reference. The contents of the block-reference-bundle is generated out of the dev sources and is normally handed over to the doc only once, at the end of a release cycle.

No, this shouldn't be a heading. IMO, code markup would be more appropriate. See also the (now obsolete) doc at https://docbuilder1.eur.ad.sag/jenkins/job/pab/job/trunk-webhelp_en/lastSuccessfulBuild/artifact/out/webhelp/index.html#page/pab-webhelp%2Fre_AnaBui_block_reference_Aggregates_Mean.html

But this is a change that can only be done by the dev team. @skom-sag , what do you think?

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+#### Step 7: Save the model and go back to the model manager
+
+1.  In the toolbar of the model editor, click the following icon to save your newly created model:
+    ![Save icon](/images/streaming-analytics/analytics-builder/icon-save.png)
+
+2.  In the toolbar of the model editor, click the following icon to leave the model editor and thus to return to the model manager:
+    ![Close icon](/images/streaming-analytics/analytics-builder/icon-exit.png)
+
+    {{< c8y-admon-info>}}
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/getting-started.md:121]: refers only to step 2 - so leaving everything as it is.

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+
+**To edit a model**
+
+-   On the **Models** tab of the model manager, simply click the card that is shown for the model \(but not on the toggle button for changing the state or the drop-down menu for changing the mode\).
+
+    Or click the actions menu of the card and then click **Edit**.
+
+    If the model is in the Active state, a dialog appears informing you that you can only view the model. When you click **Continue**, the model editor appears and you can view the model, but you cannot change it.
+
+    See [Using the model editor](/streaming-analytics/analytics-builder/#using-the-model-editor) for further information.
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:147]: Duplicate, removed

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+
+### Creating a model from a sample
+
+You can create a new model from each sample that is currently listed in the model manager. The new model gets the same name, description and tags as the sample.
+
+{{< c8y-admon-info>}}
+You must save the new model so that it is listed in the model manager. If a model with that name already exists, you are prompted to save the new model with a different name.
+{{< /c8y-admon-info>}}
+
+**To create a model from a sample**
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:301]: Changed to 4th level (not 5th).

File: content/streaming-analytics/block-reference-bundle/aggregate.md [hunk 13]
```diff
@@ -0,0 +1,985 @@
...
+<td><span>float</span>
+</td>
+</tr>
+</tbody>
+</table>
+
+
+### Counter
+
+**apama.analyticskit.blocks.core.Counter**
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/block-reference-bundle/aggregate.md:192]: See above

File: content/streaming-analytics/block-reference-bundle/aggregate.md [hunk 14]
```diff
@@ -0,0 +1,985 @@
...
+<td><span>float</span>
+</td>
+</tr>
+</tbody>
+</table>
+
+
+### Discrete Statistics
+
+**apama.analyticsbuilder.blocks.DiscreteStatistics**
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/streaming-analytics/block-reference-bundle/aggregate.md:268]: See above for all of the following block reference comments. I'll set all other comments below to Resolved.

**Pull request merged**